rules/macos/persistence_credential_access_authorization_plugin_creation.toml

```diff
@@ -41,7 +42,7 @@ name = "Boot or Logon Autostart Execution"
 reference = "https://attack.mitre.org/techniques/T1547/"
 [[rule.threat.technique.subtechnique]]
 id = "T1547.002"
-name = "Authentication Package"
+name = "Boot or Logon Autostart Execution: Authentication Package"
```

Comment:
Will be correct when https://github.com/elastic/detection-rules/pull/2073 is merged